kmain.py

```diff
@@ -76,32 +160,16 @@
 
 print(model.summary())
 
-print("Would you like to restore a previously saved model? (y/n)")
-choice = input()
-
-if (choice=='y' or choice=='Y'):
-  #path = input("Enter path: ")
-  model = load_model("checkpoints/best_model13.h5")
-
-print("\n")
 
+train_flag = train_test()
 
-train_flag = True
+df = load_csv(train_flag)
 
-print("Train? (y for train, n for test)")
-choice = input()
-if (choice =='n' or choice=='N'):
-  df = pd.read_csv("adult/adult_test.csv")
-  BATCH_SIZE = df.shape[0]
-  EPOCHS = 1
-  train_flag = False
-  
-else:
-  df = pd.read_csv("adult/adult_train.csv")
+print("Would you like to load previous saved model (y/n): ")
+model_choice = input()
+if (model_choice=='y' or model_choice=='Y'):
+  model = load_model("model.h5")
```

Comment:
Is this not inconsistent with [Line No 189](https://github.com/americast/algo_bias/pull/13/files#diff-0d2eb74693a923211c75e8d685eb25f5a6e02866f3598b5499d426f5eebd1701R189)?

Reply:
I forgot to change that, thanks for the reminder.